app/src/main/java/de/madem/homium/repositories/InventoryRepository.kt

```diff
@@ -0,0 +1,13 @@
+package de.madem.homium.repositories
+
+import de.madem.homium.models.InventoryItem
+import kotlinx.coroutines.flow.Flow
+
+interface InventoryRepository {
+
+    fun getAllInventoryItems(): Flow<List<InventoryItem>>
+
+    //TODO: Add return type
+    fun deleteInventoryItemById(id: Int)
```

Comment:
Please make this Function a `suspend`-Function and call it in `viewModelScope.launch`

Reply:
Resolved